supertokens_python/recipe/session/interfaces.py

```diff
@@ -450,7 +449,9 @@ async def get_claim_value(
 
     @abstractmethod
     async def remove_claim(
-        self, claim: SessionClaim[Any], user_context: Union[Dict[str, Any], None] = None
+        self,
+        claim: SessionClaim[_T],  # pyright: ignore[reportInvalidTypeVarUse]
```

Comment:
why did python complain about `reportInvalidTypeVarUse`?

Reply:
we aren't doing anything with `_T` (not returning it) so pyright isn't happy. Change it to `claim: SessionClaim[Any]`